Pipfile

```diff
@@ -32,6 +32,9 @@ graphql-core = ">=2.1,<3"
 pytz = ">=2017.2"
 six = ">=1.10.0"
 sqlalchemy = ">=1.3.0,<2"
+neo4j = "==1.7.4"
```

Comment:
General question: I notice that some of these other ones don't specify a particular version but rather a set of possible versions. How big a deal is this? I remember being told in some of the Github issues that some functionality that we rely on (e.g. the output format for RedisGraph) would change in future releases.